### ISSUE TITLE
PDF/Docx links on Video Tutorials section are missed

### ISSUE THREAD

**Issue by GlowingCrystallineEntity** [coursera-dl/coursera-dl#461]
I'm using the latest version of coursera-dl, python 3, on Windows.  Often the videos have pdf or docx links associated with them, with links just to the left of the subtitle and video links.  Your tool doesn't seem to d/l those.

**Comment by balta2ar** [comment]
Please see "Filing an issue/Reporting a bug" (https://github.com/coursera-dl/coursera-dl#filing-an-issuereporting-a-bug) section on what information you need to provide when filing an issue.

**Comment by GlowingCrystallineEntity** [comment]
I did.  There's nothing more I can add that would be of any use to the developers in reproducing this issue.

**Comment by balta2ar** [comment]
How about name of the course where you're experiencing the issue?
How about command line that you're using?
How about links to the exact "pdf or docx" that are not downloaded?
How about links to pages within the course where these missing links are present?

**Comment by GlowingCrystallineEntity** [comment]
Well… it looks like the issue is that you're renaming those files, when they already have human-readable names, so I wasn't finding them when searching to verify the download.  But -- I think! as I've already grabbed them with a generic downloader -- they were actually being downloaded.  So, my bad.

As for the rest -- I didn't include it because there's nothing about it that's notable.  I described where the links are, e.g.; and I'm using the most generic command in your "Running The Script" example; these links are hardly specific to any one course, and I'd rather not post my own course names.

Anyway, great script, thanks for working on it. :)
